tests/tests_suite_test.go

```diff
@@ -55,6 +60,30 @@ var _ = JustAfterEach(func() {
 	}
 })
 
+// installTestsNetworkPolicy install NetworkPolicy delay all ingress / egress traffic in the given namespace.
+// Installing deny all network policy provides coverage for kubemacpool network-policy in e2e tests.
+func installTestsNetworkPolicy(name, ns string) error {
+	np := &netv1.NetworkPolicy{
+		ObjectMeta: metav1.ObjectMeta{Name: name},
+		Spec: netv1.NetworkPolicySpec{
+			PodSelector: metav1.LabelSelector{},
```

Comment:
Can you limit is to only the kubemcapool namespace?

Reply:
The network-policy is create in kubemacpool namespaces and doesnt affect other namespaces.